x-pack/plugins/lists/server/services/exception_lists/create_endoint_event_filters_list.ts

```diff
@@ -1,82 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { SavedObjectsClientContract } from 'kibana/server';
-import uuid from 'uuid';
-import { Version } from '@kbn/securitysolution-io-ts-types';
-import type { ExceptionListSchema } from '@kbn/securitysolution-io-ts-list-types';
-import { getSavedObjectType } from '@kbn/securitysolution-list-utils';
-import {
-  ENDPOINT_EVENT_FILTERS_LIST_DESCRIPTION,
-  ENDPOINT_EVENT_FILTERS_LIST_ID,
-  ENDPOINT_EVENT_FILTERS_LIST_NAME,
-} from '@kbn/securitysolution-list-constants';
-
-import { ExceptionListSoSchema } from '../../schemas/saved_objects';
-
-import { transformSavedObjectToExceptionList } from './utils';
-
-interface CreateEndpointEventFiltersListOptions {
-  savedObjectsClient: SavedObjectsClientContract;
-  user: string;
-  tieBreaker?: string;
-  version: Version;
-}
-
-/**
- * Creates the Endpoint Trusted Apps agnostic list if it does not yet exist
```

Comment:
Doesn't this already do what the PR says? Create the list if it doesn't exist, else return a `null` when there's a conflict (I presume when there's a duplicate). 🤔 

Or that this doesn't check the list content but only list id's that may be different but have duplicate content. Isn't that what you're trying to avoid here?

Reply:
You're right, the PR description is outdated. I will re write it

Reply:
Sorry, it was already updated. Look at the **Update** section in description. We don't want to create the list from the manifest as it would be created even if the user don't want to use event filters. Removing this code the list will be created only when the user wants to use event filters from the UI